### PR TITLE
schedule update, 4 invited talks

### PR DESCRIPTION
**Pull request by samumantha**
Updated schedule to include 4 invited talks each 20 min, 2 breaks 10 min each after every hour, one Q&A session after all invited talks and depending on how long this takes a longer or shorter intro to nordicRSE
Then a short intro to the format and how to contribute, hackmd and how it will go. and a first short unconference session

Discussion open now:

### CODE REVIEW

**Comment by bast** [comment]
I like it!

**Comment by PatricHolmvall** [comment]
Good update, I especially like that there are now two breaks.

**Comment by samumantha** [comment]
Should there also be two official break on day 2? 
Maybe the unconference theme during whole time is enough to have only one 'official' break?

**Comment by PatricHolmvall** [comment]
> Should there also be two official break on day 2?
> Maybe the unconference theme during whole time is enough to have only one 'official' break?

Great question. I feel that the theme is good enough to only have one official break, and people can then take breaks as they need. Are there any drawbacks of this? Is it less fair to people presenting during this longer session?

File: content/events/2021-online-unconference/_index.md [hunk 1]
```diff
@@ -56,19 +56,25 @@ By submitting a contribution you agree
...
-  - 13:10 : Invited talks and NordicRSE
-  - 14:20 : Break
-  - 14:40 : Unconference session
+  - 13:00 : Welcome 
+  - 13:10 : Invited talk I
+  - 13:30 : Invited talk II
+  - 13:50 : Break
+  - 14:00 : Invited talk III
+  - 14:20 : Invited talk IIII
+  - 14:40 : Q&A and nordicRSE
```
Comment (PatricHolmvall) [inline content/events/2021-online-unconference/_index.md:65]:
```suggestion
  - 14:40 : Q&A and Nordic-RSE
```

**Pull request merged**